backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/DemoAuthenticationConfiguration.java

```diff
@@ -168,7 +179,16 @@ public List<OrganizationRoleClaims> findAllOrganizationRoles() {
       String username = Optional.ofNullable(_getCurrentUser.get()).orElseThrow().getUsername();
       Optional<OrganizationRoleClaims> claims =
           _oktaRepo.getOrganizationRoleClaimsForUser(username);
-      return claims.isEmpty() ? List.of() : List.of(claims.get());
+      List<OrganizationRoleClaims> oktaOrgRoleClaims =
+          claims.isEmpty() ? List.of() : List.of(claims.get());
+      if (!isSiteAdmin()) {
+        if (_featureFlagsConfig.isOktaMigrationEnabled()) {
```

Comment:
could we unnest this second if and put it together with 184?

Reply:
Yes! Will do this!

Reply:
Done!